examples/Clustering.ipynb

```diff
@@ -116,14 +118,12 @@
OmgLFYDC0tLRgaGoLL5Vrr4VQ1Yq4Wh5ivhSPmanGI+Vo4Yq7mR1EUxONxNDU1QaebOxJl3VtUdDodNmzYsNbDuCZcLpf4EC8QMVeLQ8zXwhFztTjEfC0cMVdzM58lRUUE0woEAoFAIKhahFARCAQCgUBQtQihsoaYzWZ86lOfgtlsXuuhVD1irhaHmK+FI+ZqcYj5WjhirpaHdR9MKxAIBAKB4PpFWFQEAoFAIBBULUKoCAQCgUAgqFqEUBEIBAKBQFC1CKEiEAgEAoGgahFCRSAQCAQCQdUihMoak81msXPnTkiShGPHjs3Yd+LECbz2ta+FxWJBS0sLPv/5z6/NINeQgYEBvO9970NHRwesViu6urrwqU99CrlcbsZxYq40/vEf/xHt7e2wWCzYv38/fvvb3671kNacJ598Eq95zWvgdDpRX1+Pt7zlLejt7Z1xTCaTwWOPPQafzweHw4G3v/3tGBsbW6MRVxef/exnIUkSPvzhD09vE/OlMTIygocffhg+nw9WqxXbtm3Dq6++Or1fURT85V/+JRobG2G1WnHPPffgwoULazji9YUQKmvMxz72MTQ1Nc3aHovFcN9996GtrQ2HDx/GU089hU9/+tP42te+tgajXDvOnTsHWZbx1a9+FadPn8YXv/hFfOUrX8H/+B//Y/oYMVca3/3ud/GRj3wEn/rUp3DkyBHs2LED999/P8bHx9d6aGvKCy+8gMceewwvvfQSDh48iHw+j/vuuw/JZHL6mP/+3/87fvzjH+N73/seXnjhBYyOjuJtb3vbGo66OnjllVfw1a9+Fdu3b5+xXcwXCYfDuP3222E0GvHMM8/gzJkz+Nu//Vt4PJ7pYz7/+c/jH/7hH/CVr3wFL7/8Mux2O+6//35kMpk1HPk6QhGsGT/96U+VzZs3K6dPn1YAKEePHp3e90//9E+Kx+NRstns9LaPf/zjSk9PzxqMtLr4/Oc/r3R0dEw/FnOlsW/fPuWxxx6bflwsFpWmpiblySefXMNRVR/j4+MKAOWFF15QFEVRIpGIYjQale9973vTx5w9e1YBoBw6dGithrnmxONxZePGjcrBgweVO++8U/nTP/1TRVHEfJXy8Y9/XLnjjjvm3C/LsuL3+5WnnnpqelskElHMZrPy7W9/ezWGuO4RFpU1YmxsDI8++ii+8Y1vwGazzdp/6NAhvO51r4PJZJredv/996O3txfhcHg1h1p1RKNReL3e6cdirkgul8Phw4dxzz33TG/T6XS45557cOjQoTUcWfURjUYBYPpzdPjwYeTz+Rlzt3nzZrS2tt7Qc/fYY4/hoYcemjEvgJivUn70ox9h7969eOc734n6+nrs2rUL//zP/zy9/9KlSwgGgzPmqqamBvv377/h5mqpCKGyBiiKgkceeQQf+MAHsHfv3orHBINBNDQ0zNimPg4Ggys+xmqlr68PX/rSl/BHf/RH09vEXJHJyUkUi8WKc3EjzcPVkGUZH/7wh3H77bdj69atAPg5MZlMcLvdM469kefuO9/5Do4cOYInn3xy1j4xXxoXL17El7/8ZWzcuBHPPvssPvjBD+JP/uRP8PWvfx2A9h0k/i+XjhAqy8gnPvEJSJI078+5c+fwpS99CfF4HI8//vhaD3nNWOhclTIyMoIHHngA73znO/Hoo4+u0cgF653HHnsMp06dwne+8521HkrVMjQ0hD/90z/Ft771LVgslrUeTlUjyzJ2796Nz3zmM9i1axfe//7349FHH8VXvvKVtR7adYNhrQdwPfHRj34UjzzyyLzHdHZ24vnnn8ehQ4dmNarau3cvfu/3fg9f//rX4ff7Z0XQq4/9fv+yjnstWOhcqYyOjuL1r389brvttllBstf7XC2U2tpa6PX6inNxI83DfHzoQx/Cf/zHf+CXv/wlNmzYML3d7/cjl8shEonMsBLcqHN3+PBhjI+PY/fu3dPbisUifvnLX+J//+//jWeffVbM1xUaGxuxZcuWGdtuuukm/OAHPwCgfQeNjY2hsbFx+pixsTHs3Llz1ca5rlnrIJkbkcHBQeXkyZPTP88++6wCQPn+97+vDA0NKYqiBYjmcrnp5z3++OM3ZIDo8PCwsnHjRuVd73qXUigUZu0Xc6Wxb98+5UMf+tD042KxqDQ3N9/wwbSyLCuPPfaY0tTUpJw/f37WfjU49Pvf//70tnPnzt2QwaGKoiixWGzGd9TJkyeVvXv3Kg8//LBy8uRJMV8l/O7v/u6sYNoPf/jDyq233qooihZM+4UvfGF6fzQaFcG0i0AIlSrg0qVLs7J+IpGI0tDQoLznPe9RTp06pXznO99RbDab8tWvfnXtBroGDA8PK93d3cob3vAGZXh4WAkEAtM/KmKuNL7zne8oZrNZ+dd//VflzJkzyvvf/37F7XYrwWBwrYe2pnzwgx9UampqlF/84hczPkOpVGr6mA984ANKa2ur8vzzzyuvvvqqcuutt04vNgJlRtaPooj5Uvntb3+rGAwG5W/+5m+UCxcuKN/61rcUm82mfPOb35w+5rOf/azidruV//t//69y4sQJ5Xd+53eUjo4OJZ1Or+HI1w9CqFQBlYSKoijK8ePHlTvuuEMxm81Kc3Oz8tnPfnZtBriGPP300wqAij+liLnS+NKXvqS0trYqJpNJ2bdvn/LSSy+t9ZDWnLk+Q08//fT0Mel0Wvlv/+2/KR6PR7HZbMpb3/rWGYL4RqdcqIj50vjxj3+sbN26VTGbzcrmzZuVr33tazP2y7Ks/MVf/IXS0NCgmM1m5Q1veIPS29u7RqNdf0iKoiir7m8SCAQCgUAgWAAi60cgEAgEAkHVIoSKQCAQCASCqkUIFYFAIBAIBFWLECoCgUAgEAiqFiFUBAKBQCAQVC1CqAgEAoFAIKhahFARCAQCgUBQtQihIhAIBAKBoGoRQkUgEAgEAkHVIoSKQCAQCASCqkUIFYFAIBAIBFXL/w+sPPs/fBzEswAAAABJRU5ErkJggg==",
```

Comment:
Add this in the images dir instead of inline.

Reply:
@ibigio i didn't make this notebook originally but it seems like this line is from the matplotlib, not directly inline